packages/docs/components/TableOfContents/api.tsx

```diff
@@ -96,6 +100,10 @@ export const TableOfContents: React.FC = () => {
           <strong>@remotion/layout-utils</strong>
           <div>Layout helpers</div>
         </TOCItem>
+        <TOCItem link="/docs/animation-utils">
+          <strong>@remotion/animation-utils</strong>
+          <div>Keyframing utilities</div>
+        </TOCItem>
```

Comment:
Oops sorry I duplicated it

```suggestion
```

Reply:
Should I use `Keyframing Utilities` or `Utilities for animation` is fine?